improver/ensemble_calibration/ensemble_calibration.py

```diff
@@ -642,27 +614,13 @@ def compute_initial_guess(
         of the forecast predictor and the truth, if requested. Otherwise,
         default values for the coefficients will be used.
 
-        If the predictor_of_mean_flag is "mean", then the order of
-        the initial_guess is [gamma, delta, alpha, beta]. Otherwise, if the
-        predictor_of_mean_flag is "realizations" then the order of the
-        initial_guess is [gamma, delta, alpha, beta0, beta1, beta2], where
-        the number of beta variables will correspond to the number of
-        realizations. In this example initial guess with three beta
-        variables, there will correspondingly be three realizations.
-
-        The coefficients relate to adjustments to the ensemble mean or the
-        ensemble realizations, and adjustments to the ensemble variance:
-
-        .. math::
-            alpha + beta * ensemble\\_mean
-
-        or
-
-        .. math::
-            alpha + beta0 * realization0 + beta1 * realization1
-
-        .. math::
-            gamma + delta * ensemble\\_variance
+        If the predictor is "mean", then the order of the initial_guess is
+        [gamma, delta, alpha, beta]. Otherwise, if the predictor is
+        "realizations" then the order of the initial_guess is
+        [gamma, delta, alpha, beta0, beta1, beta2], where the number of beta
+        variables will correspond to the number of realizations. In this
+        example initial guess with three beta variables, there will
+        correspondingly be three realizations.
 
         The default values for the initial guesses are in
         [gamma, delta, alpha, beta] ordering:
```

Comment:
The bullet points below don't work properly

Reply:
I've amended these bullet points.